README.md

```diff
@@ -32,6 +32,13 @@ The repository has the following permanent branches:
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
 
+## Screenshots
```

Comment:
Screenshots should not be added to the repo. Adding non binary files unnecessarily increases the size of the repository which is not very easy to reverse.
There is a better way of doing this.

Reply:
@isabelcosta 
commit titled `docs: Add screenshots to README.md` by `IlyaMaie` shouldn't be merged into any other branch.

Reply:
I have a suggestion @m-murad , I can do it later today (if no one does it until then), put the images in a github issue and upload them to Github, and replace the links to the relative path of the images for the URL from the images uploaded. This will still make this documentation available on the README.
About the git commit which was already done, is there a way, we can remove the commit from this branch / PR (exclude the single commit)? @annabauza @m-murad @mayburgos @robotjellyzone @CheshtaK @Buddhiprabha

Reply:
Images/GIFs should be uploaded to a GitHub issue or PR as attachments. Then the link of the assets can be copied from there. and includes in the readme.md.

Reply:
You can google 'git rebase interactive'. This will give you some idea.